web/src/Proposal.jsx

```diff
@@ -1,26 +1,15 @@
 import React from "react";
```

Comment:
NP: I'd rename this component to ProposalActions or similar.

Reply:
Yes. And I would change the wording about storage in general: e.g., replacing "Target" with "Partitioning" or something like that. Let's do that in an open PR.